source/index.html.md

```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
 }
 ```
 
+## Organization Endpoint
+You can upload organization data to Shipamax. Organization data can be retrieved by ID, or by a query filter. You can also update an organization's details and delete an organization by ID.
+
+### POST
+Using the following endpoint you can create a new organization, you will need to provide a JSON body with all of the required details. Organizations are unique to their `externalIds`.
+
+https://public.shipamax-api.com/api/v2/Organizations
+
+>Example JSON body to create an organization
+```json
+{
+    "externalId": "A",
+    "carrier": false,
+    "consignee": true,
+    "creditor": true,
+    "forwarder": false,
+    "debtor": false,
+    "shipper": false,
+    "active": false,
+    "updated": "2020-01-01"
+}
+```
+
+>The POST endpoint will respond with JSON like this:
+```json
+{
+  "id": 9,
+  "externalId": "FF",
+  "carrier": false,
+  "consignee": true,
+  "creditor": true,
+  "forwarder": false,
+  "debtor": false,
+  "shipper": false,
+  "active": false,
+  "updated": "2020-01-01T00:00:00.000Z",
+  "companyId": 100000
```

Comment:
I think we can use the "hidden" property of the LB3 JSON schema to make this not return if you want.